### ISSUE TITLE
Support PlantUML Server as renderer

### ISSUE THREAD

**Issue by nalaka** [mikitex70/plantuml-markdown#19]
Is it possible to use a locally running [PlantUML Server](https://github.com/plantuml/plantuml-server) for rendering instead of the `plantuml` executable? Maybe a setting to choose the executable or server?

Using the server is way faster than the binary.
The difference is highly noticeable when using this plugin from something like MkDocs with live reload.

For comparison, this [Visual Code Plugin](https://github.com/qjebbs/vscode-plantuml) allows toggling the renderer.

Note: A PlantUML server can be run at port 8080 with `docker run -d -p 8080:8080 plantuml/plantuml-server:jetty`

**Comment by mikitex70** [comment]
I can work on it only in the weekend.
Give me some time and I will try to implement your request.
Thanks for your suggestion.

**Comment by mikitex70** [comment]
I've a working version using a plantuml server but... I need to change the plugin name to avoid a module name conflict.
Do you think will be reasonable to change the name of the plugin from `plantuml` to `plantuml-markdown`?
This means that the configurations in other softwares (like MkDocs) must be changed too.

**Comment by nalaka** [comment]
I think it should be fine.

And thanks for the super fast response - I must say I am pleasently surprised!

:)

**Comment by mikitex70** [comment]
Released version `2.0.0` which implements your request.
Please update the plugin name in MkDocs from `plantuml` to `plantuml-markdown`.
See the `README.md` for configuration options.

**Comment by nalaka** [comment]
> Released version `2.0.0` which implements your request.
> Please update the plugin name in MkDocs from `plantuml` to `plantuml-markdown`.
> See the `README.md` for configuration options.

Thank you very much. It works perfectly.

I updated `mkdocs.yml` to

```
- plantuml_markdown:
        server: http://localhost:8080  # PlantUML server, for remote rendering
```

And started the PlantUML server with Docker:

```
docker run -d -p 8080:8080 plantuml/plantuml-server:jetty
```
